content/gettingstarted/index.adoc

```diff
@@ -39,89 +39,66 @@ icon:lightbulb[size=1x]{nbsp}For several commands listed on this page, the OpenS
+It will pull the latest version of Kiali Operator and Kiali itself, and install them with default options. If Kiali was already installed, you will be prompted about removing it first.
 
-icon:lightbulb[size=1x]{nbsp} The Kiali Operator install script can be configured by setting environment variables when running it. For example, if you do not want the install script to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself, as explained above), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script. Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available to you.
+icon:lightbulb[size=1x]{nbsp} The Kiali Operator install script can be configured by setting environment variables when running it. For example, if you do not want the install script to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script. Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available to you.
 
 icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
 
-To install the Kiali Operator (and, by default, the Kiali CR), run the install script via this command:
+During the installation, you will be prompted for the authentication strategy you want to use among `login`, `anonymous` and `openshift`.
+See the link:#_login_options[Login Options] section below for more information.
+
+=== Configure Kiali
+
+The Kiali Operator runs inside the `kiali-operator` namespace and listens for the creation of a Kiali CR (custom resource) object. When it sees one created, it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file].
+
```

Comment:
In addition to a created file I think it also responds to an edit?  Also. maybe mention that the pod will be restarted after the changes are applied?

Reply:
I've just rewritten as "... and listens for a Kiali CR (custom resource) creation or changes.", not saying too much to keep it short.